nbextensions/bulkImportCell/tabs/multiAppInfo.js

```diff
@@ -0,0 +1,158 @@
+define([
+    'bluebird',
+    'common/html',
+    'common/ui',
+    './fileTypePanel',
+    'common/cellComponents/tabs/infoTab',
+], (Promise, html, UI, FileTypePanel, InfoTab) => {
+    'use strict';
+    const cssBaseClass = 'kb-bulk-import-info';
+
+    /**
+     * This widget is responsible for providing info for each of the bulk import apps.
+     *
+     * It maintains the state of the currently shown app info (by file type)
```

Comment:
this should really be changed to index by app ID instead of file type -- if we go on to create a bulk app runner, the concept of file type doesn't really apply there.

Reply:
That'll need to be a change when we get to that step. Same for the "fileType" panel - should be for apps, too. Or maybe that should be some form of class hierarchy / interface?